exosphere-shared/example-apps/docker-service/application.yml

```diff
@@ -0,0 +1,11 @@
+name: docker-app
+description: An application that relies on dockerhub images
+version: 0.0.1
+
+services:
+  public:
+    local-service:
+      docker_image: exospheredev/local-service
+      location: ./local-service
```

Comment:
What's the point of defining `docker_image` as well then? Isn't the image name generated in `exo-setup`? Having that in two places might be confusing. It seems like it's not being used here anyways?